public/app/features/browse-dashboards/components/DeleteProvisionedFolderForm.tsx

```diff
@@ -67,8 +69,8 @@ function FormContent({
   useEffect(() => {
     if (request.isSuccess && repository) {
       if (workflow === 'branch') {
-        // TODO: handle display banner https://github.com/grafana/git-ui-sync-project/issues/300
-        // TODO: implement when BE is ready
+        const prUrl = request.data?.urls?.newPullRequestURL;
+        navigate(`/dashboards?new_pull_request_url=${prUrl}`);
```

Comment:
Nit: do we need to also check that `prUrl` actually exists (i.e., is not undefined or empty string)?